doc/gtdb.md

```diff
@@ -0,0 +1,137 @@
+
+This tutorial will explain various strategies of utilizing GTDB taxonomy / phylogeny in a meta'omics analysis with Woltka.
+
+
+## Contents
+
+- [Woltka directly parses GTDB taxonomy](#woltka-directly-parses-gtdb-taxonomy)
+- [Reformat GTDB as taxdump style](#reformat-gtdb-as-taxdump-style)
+- [Build GTDB genome database](#build-gtdb-genome-database)
+- [Use GTDB phylogeny](#use-gtdb-phylogeny)
+- [Use WoL database with GTDB taxonomy](#use-wol-database-with-gtdb-taxonomy)
+
+
+## Woltka directly parses GTDB taxonomy
+
+The GTDB taxonomy ([`bac120_taxonomy_r89.tsv`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/bac120_taxonomy_r89.tsv) and [`ar122_taxonomy_r89.tsv`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/ar122_taxonomy_r89.tsv)) is provided in the format of **Greengenes-style lineage strings**. For example: _Escherichia coli_ O157:H7 str. Sakai (NCBI RefSeq assembly: [GCF_000008865.1](https://www.ncbi.nlm.nih.gov/assembly/GCF_000008865.1/)) is classified as:
+
+```
+RS_GCF_000008865.1	d__Bacteria;p__Proteobacteria;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Escherichia;s__Escherichia dysenteriae
+```
+
+This format can be directly parsed by Woltka:
+
+```bash
+woltka classify --lineage bac120_taxonomy_r89.tsv ...
+```
+
+If you work with both Bacteria and Archaea (this is the common case), you may simply concatenate the two taxonomy files before use:
+
+```bash
+cat bac120_taxonomy_r89.tsv ar122_taxonomy_r89.tsv > gtdb_taxonomy.tsv
+```
+
+Note: The [GTDB-Tk](https://github.com/Ecogenomics/GtdbTk) database ([`gtdbtk_data.tar.gz`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/gtdbtk_r89_data.tar.gz)) already provides a combined `gtdb_taxonomy.tsv` file.
+
+
+## Reformat GTDB as taxdump style
+
+Alternatively, you may consider converting GTDB taxonomy into the format of NCBI taxdump. This will allow you to connect GTDB to more bioinformatics tools such as BLAST and Kraken2 which favor the taxdump format. This will also make the Woltka output slimer, if that's your preference. For example, instead of the long lineage strings like `d__Bacteria;p__Proteobacteria;...`, you will get taxon names like `Escherichia` as the feature name.
+
+Three good things of the GTDB taxonomy (in addition to many others) are that:
+
+1) All lineages have exactly seven ranks (`d`omain, `p`hylum, `c`lass, `o`rder, `f`amily, `g`enus and `s`pecies).
+2) All ranks are filled (i.e., no "gaps" such as `g__;` as multiple other systems do).
+3) At each rank, all taxon names are unique (but duplicate names are allowed at different ranks).
+
+These characteristics make it safe to collapse GTDB lineages into taxonomic units and index them in a way that resembles NCBI taxdump (I mean, in case you are thinking about whether this trick also applies to other taxonomies).
```

Comment:
I'm not sure what you are trying to say in the parenthesis. Could you clarify it?

Reply:
I reworded. How does it sound now?